Microsoft.Toolkit.Uwp.UI.Controls.Primitives/Microsoft.Toolkit.Uwp.UI.Controls.Primitives.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+	<PropertyGroup>
+		<TargetFrameworks>uap10.0.17763</TargetFrameworks>
+		<Title>Windows Community Toolkit Primitive Controls</Title>
+		<Description>
+			This library provides controls for use in XAML which don't provide their own style. It is part of the Windows Community Toolkit.
+
+			Controls:
+			- AdaptiveGridView: Presents items in an evenly-spaced set of columns to fill the total available space.
+			- DockPanel: Define areas where you can arrange child elements either horizontally or vertically, relative to each other.
+			- StaggeredPanel: Layout of items in a column approach where an item will be added to whichever column has used the least amount of space.
+			- SwitchPresenter: A presenter which can act as a switch statement for layout providing alternate layouts based on a value.
+			- UniformGrid: Presents items in an evenly-spaced set of rows or columns to fill the total available display space.
+			- WrapPanel: Positions child elements in sequential position from left to right and breaks content to the next line.
+		</Description>
+		<PackageTags>UWP Toolkit Windows Controls XAML WrapPanel Adaptive DockPanel StaggeredPanel Staggered SwitchPresenter UniformGrid Uniform Grid</PackageTags>
+		<!-- ARM64 builds for managed apps use .NET Native. We can't use the Reflection Provider for that. -->
+		<EnableTypeInfoReflection Condition="'$(Configuration)' == 'Debug'">false</EnableTypeInfoReflection>
+		<LangVersion>8.0</LangVersion>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\Microsoft.Toolkit.Uwp.UI\Microsoft.Toolkit.Uwp.UI.csproj" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<None Include="VisualStudioToolsManifest.xml" Pack="true" PackagePath="tools" />
+		<None Include="$(OutDir)\Design\$(MSBuildProjectName).Design*.dll;$(OutDir)\Design\$(MSBuildProjectName).Design*.pdb" Pack="true" PackagePath="lib\$(TargetFramework)\Design" />
```

Comment:
Instead of `Design*`, we can use `DesignTools` as we are not relying on older system anymore. But it's okay now as I still have some cleanup to do with the Design projects.

I'll take it up as part of my next PR. so don't worry about it.

Reply:
I figured we were going to make that change but I did not want to jump the gun. Thanks @Nirmal4G C: